how-to/workspace-platform-starter/client/src/framework/bootstrapper.ts

```diff
@@ -201,8 +202,22 @@ export async function init(): Promise<boolean> {
 		authProvider.subscribe("logged-in", async () => {
 			// what behavior do you want to do when someone logs in
 			// potentially the inverse if you hid something on session expiration
-			await fireLifecycleEvent(platform, "auth-logged-in");
+			await fireLifecycleEvent(platform, "auth-logged-in", {
+				user: await authProvider.getUserInfo()
+			} as LoggedInLifecyclePayload);
 		});
+
+		// If the user is already logged in then we must fire the lifecycle event immediately
+		const isAuthenticationRequired = await authProvider.isAuthenticationRequired();
+		if (!isAuthenticationRequired) {
+			// Fire the event on the next cycle to allow the bootstrapping process to complete
+			setTimeout(async () => {
```

Comment:
Same rule about whether to include the user. Would this still suffer from late subscribers. I.e. if I subscribe to this event in a home integration does the lifecycle module already cache the last event?